src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -168,10 +177,85 @@ static aafConfig_t aafLUT42605[AAF_CONFIG_COUNT] = {  // see table in section 5.
     [AAF_CONFIG_1962HZ] = { 63, 3968,  3 }, // 995 Hz is the max cutoff on the 42605
 };
 
+static void setUserBank(const extDevice_t *dev, const uint8_t user_bank)
```

Comment:
(personally, I'd prefer to encode bank in register and make bank selection automatic (with caching). But it is not related to this PR)